Add fixture 'eurolite/led-tl-3-rgb-uv'

FloEdelmann opened the pull request:
* Add fixture 'eurolite/led-tl-3-rgb-uv'

### Fixture warnings / errors

* eurolite/led-tl-3-rgb-uv
  - :x: Mode '4-1-channel' should have 1 channels according to its name but actually has 4.
  - :x: Mode '4-1-channel' should have 1 channels according to its shortName but actually has 4.


Thank you @barnabycollins!

barnabycollins commented:
DMX specsheet can be found at https://www.steinigke.de/download/51915445-Manual-128720-1.100-eurolite-led-tl-3-rgb+uv-trusslight-de_en.pdf on pages 26-27!

barnabycollins commented:
Having had time to think about this, am I right in thinking the default values should in fact be 255 for intensities so the light is on when located / defaulted?

FloEdelmann commented:
Thanks for your contribution! I've added a few more details and renamed the mode to make our test happy. I think it's still easy to recognize.

It's not clearly defined what the default value should be (see also [#354](https://togithub.com/OpenLightingProject/open-fixture-library/issues/354), so let's leave it like it is for now.

barnabycollins commented:
Wow, your changes look great; thank you so much! And no worries; happy to help :)
